live/root/.xinitrc

```diff
@@ -1 +1,7 @@
 icewm-session -c /etc/icewm/preferences.yast2
```

Comment:
This `icewm-session` call is actually blocking, the `xset` calls are not executed. Only when exiting the X session...

After moving the `icewm-session` to the end it works fine, `xset q` reports "DPM is Disabled".